Ported over to Laravel 4

sp0tteh opened the pull request:
Hey,

I needed this today as I was sick of adding temp routes so I have quickly ported this over to Laravel 4. 
Have simply added a tag for the Laravel 3 version as I didn't really have time to make it work in both.

To test it out you can following the updated readme, you will also need to add :

```
"repositories": [
    {
            "type": "vcs",
            "url":  "git://github.com/allmyitjason/laravel-console.git"
    }
],
```

into your composer.json file until the package is submitted to [Packagist](https://packagist.org/).

Enjoy :)

sp0tteh committed:
- Ported to L4

sp0tteh committed:
Fixed package name

sp0tteh committed:
Fixed routes and re-added white-list filter

sp0tteh committed:
Updated docs for Laravel 4

sp0tteh committed:
Update README.md

sp0tteh committed:
Update README.md

tortuetorche commented:
:+1:  Good job Jason !

darsain commented:
Sorry dude, I see this. But I'm not gonna merge it from several reasons.

One is that I don't know [how to address backwards compatiblity with L3 `artisan install`](https://github.com/Darsain/laravel-console/issues/10#issuecomment-21403356).

The other is that I already have a working L4 version on my localhost, which does some re-factoring and improvements with error handling in L4. I left this open so others can see that it is here, and they won't waste their time submitting PRs.

ajimix committed:
Added command + enter shortcut

As the interface says, you can use command + enter to submit but wasn't working.

sp0tteh committed:
Merge pull request #1 from ajimix/patch-1

Added command + enter shortcut

sp0tteh committed:
Fix issue with phpunit including multiple times and throwing re-declare errors

darsain commented:
My rewrite is now in master, but it is not yet really usable. Error handling (#13) just doesn't work and I can't figure out how to fix it.
